src/main/java/service/impl/InputHandler.java

```diff
@@ -0,0 +1,32 @@
+package service.impl;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import service.InputFileHandler;
+
+public class InputHandler implements InputFileHandler {
+    @Override
+    public List<String> readFile(String fileName) {
+        try (BufferedReader bufferedReader = new BufferedReader(
+                new FileReader(fileName))) {
```

Comment:
The `fileName` parameter should be a relative path to a file. Make sure you are not using an absolute path. Everyone who pulls your project should be able to run it.